src/pages/ProgramProgressCollectionAdminPage/ProgramProcessBlock.tsx

```diff
@@ -169,7 +164,11 @@ const ProgramProcessBlock: React.VFC = () => {
                   exerciseStatus === formatMessage(pageMessages.ProgramProcessBlock.exercisePassed)
                     ? hightestScore.updatedAt
                     : ''
-
+                const examProgressPercentage =
+                  exerciseStatus === formatMessage(pageMessages.ProgramProcessBlock.exercisePassed) &&
+                  hightestScore.totalGainedPoints > 0
```

Comment:
if ths score pass the exam is 100%
unanswer or not pass is 0
check the conditional operator